hsr_dwa_planner:
  use_dwa: false

  # Robot Configuration
  acc_lim_x: 2.5
  acc_lim_y: 0.0
  acc_lim_th: 3.2
  max_vel_trans: 0.2
  min_vel_trans: 0.1
  max_vel_x: 0.2
  min_vel_x: 0.0
  max_vel_y: 0.0
  min_vel_y: 0.0
  max_vel_theta: 1.0
  min_vel_theta: 0.6

  # Goal Tolerance
  yaw_goal_tolerance: 0.17
  xy_goal_tolerance: 0.10
  latch_xy_goal_tolerance: true

  # Forwared Simulation
  sim_time: 1.7
  sim_granularity: 0.025
  vx_samples: 20
  vy_samples: 0
  vth_samples: 30
  controller_frequency: 20.0

  # Trajectory Scoring
  path_distance_bias: 32.0
  goal_distance_bias: 24.0
  occdist_scale: 0.01
  forward_point_distance: 0.325
  stop_time_buffer: 1.0
  scaling_speed: 0.25
  max_scaling_factor: 0.2
  publish_cost_grid: false
  oscillation_reset_dist: 0.05
  prune_plan: true
